docs/app/pages/components/components-sections/color/color-picker/color-picker.component.html

```diff
@@ -160,6 +160,9 @@
         Emitted when the user presses enter in the input panel text field.
         This can be used to commit a color change and/or close a popup.
     </tr>
+    <tr uxd-api-property name="inputModeChange" type="ColorPickerInputMode">
```

Comment:
I think this should be `type="'hex' | 'rgba'"` as the input above, because we don't describe what `ColorPickerInputMode` is in the documentation.